iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj

```diff
@@ -12,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\provisioning\service\src\Microsoft.Azure.Devices.Provisioning.Service.csproj" />
```

Comment:
Why is this required?

Reply:
Not sure how it got in there. I'm removing it.